editor/editor_help.cpp

```diff
@@ -564,18 +564,37 @@ void EditorHelp::_class_desc_select(const String &p_select) {
 		emit_signal("go_to_help", "class_name:" + p_select.substr(1, p_select.length()));
 		return;
 	} else if (p_select.begins_with("@")) {
+		String tag = p_select.substr(1, 6);
+		String link = p_select.substr(7, p_select.length());
+
+		String topic;
+		Map<String, int> *table = NULL;
+
+		if (tag == "method") {
+			topic = "class_method";
+			table = &this->method_line;
+		} else if (tag == "member") {
+			topic = "class_property";
+			table = &this->property_line;
+		} else if (tag == "enum  ") {
+			topic = "class_enum";
```

Comment:
Are the two spaces after `enum` expected here?

Reply:
Yes, I made the tags all with 6 characters to simplify the logic, filled with spaces at the end. Could replace this with `if (tag.begins_with("enum"))`.